inbox/quick-response.xml

```diff
@@ -0,0 +1,212 @@
+    <dl>
+      <di>
+        <dt>&lt;translation&gt;</dt>
+        <dd>One for each <tt>&lt;body&gt;</tt> and <tt>@xml:lang</tt> included in the <tt>&lt;message&gt;</tt> this response is a part of. The <tt>@value</tt> is the internationalized textual payload to put into the <tt>&lt;body&gt;</tt> of the message stanza that is sent when this response is selected. The <tt>@label</tt> is an optional internationalized textual label for this response. Clients that offer UI for quick selection of one of the allowed responses MAY refer to this response by label instead of value.</dd>
+      </di>
+    </dl>
+  </section2>
+
+  <section2 anchor="available-action-element" topic="Available Action">
+    <p>Each available action is represented by an <tt>&lt;available-action&gt;</tt> element in the <tt>&ns;</tt> namespace.</p>
+
+<example><![CDATA[<available-action xmlns="]]>&ns;<![CDATA[" id="merge">
+  <label xml:lang="en">Merge Now!</label>
+</available-action>]]></example>
+
+    <dl>
+      <di><dt>@id</dt><dd>A string identifying the action. When selected, this id is sent in an <tt>&lt;action-selected&gt;</tt> element as part of a message stanza without any <tt>&lt;body&gt;</tt> elements.</dd></di>
+      <di><dt>&lt;label&gt;</dt><dd>Internationalized textual label for this action. One with the same <tt>@xml:lang</tt> of each <tt>&lt;body&gt;</tt> included in this <tt>&lt;message&gt;</tt>.</dd></di>
+    </dl>
+  </section2>
+
+  <section2 anchor="action-selected-element" topic="Action Selection">
+    <p>A selected action is represented by an <tt>&lt;action-selected&gt;</tt> element in the <tt>&ns;</tt> namespace.</p>
+
+<example><![CDATA[<action-selected xmlns="]]>&ns;<![CDATA[" id="merge" />]]></example>
+
+    <dl>
+      <di><dt>@id</dt><dd>The id of the selected action, as defined in the selected <tt>&lt;available-action&gt;</tt>.</dd></di>
+    </dl>
+  </section2>
+</section1>
+
+<section1 anchor="protocol" topic="Protocol">
+  <section2 anchor="sending-allowed-responses" topic="Sending a Set of Allowed Responses">
+    <p>A message with allowed responses is sent by including one or more <tt>&lt;allowed-response&gt;</tt> elements with distinct values.</p>
+
+<example><![CDATA[<message from="rootbot@example.com">
+  <body xml:lang="en">Execute `rm -rf /`? (yes/no)</body>
+  <allowed-response xmlns="]]>&ns;<![CDATA[">
+    <translation xml:lang="en" value="yes" label="Sure!" />
+  </allowed-response>
+  <allowed-response xmlns="]]>&ns;<![CDATA[">
+    <translation xml:lang="en" value="no" label="Uuuuuuh..." />
+  </allowed-response>
+</message>]]></example>
+
+    <p>A single message MUST NOT contain multiple <tt>&lt;translation&gt;</tt> elements with the same values for both <tt>@xml:lang</tt> and <tt>@value</tt>. The same applies for the combination of <tt>@xml:lang</tt> and <tt>@label</tt>.</p>
+    <p>Clients that receive a message containing allowed responses can offer UI to quickly and conveniently select one of the allowed responses.</p>
```

Comment:
nit: this could probably read "MAY offer a UI to quickly"…

Reply:
I hardly see the difference between text that doesn't use 2119 at all and text that precedes everything with MAY, but I'll change it (and the other notes regarding 2119 too)